pallets/funding/src/functions.rs

```diff
@@ -174,14 +183,24 @@ impl<T: Config> Pallet<T> {
 	/// # Next step
 	/// Users will pond PLMC for this project, and when the time comes, the project will be transitioned
 	/// to the next round by `on_initialize` using [`do_evaluation_end`](Self::do_evaluation_end)
-	pub fn do_start_evaluation(caller: AccountIdOf<T>, project_id: ProjectId) -> DispatchResultWithPostInfo {
+	pub fn do_start_evaluation(
+		caller: AccountIdOf<T>,
+		project_id: ProjectId,
+		did: DID,
+		investor_type: InvestorType,
+	) -> DispatchResultWithPostInfo {
 		// * Get variables *
 		let project_metadata = ProjectsMetadata::<T>::get(project_id).ok_or(Error::<T>::ProjectNotFound)?;
 		let mut project_details = ProjectsDetails::<T>::get(project_id).ok_or(Error::<T>::ProjectDetailsNotFound)?;
 		let now = <frame_system::Pallet<T>>::block_number();
 
 		// * Validity checks *
+		// TODO: maybe check the did if we store it in project-details? if so maybe we don't need the check below
```

Comment:
If an issuer cannot evaluate their own project, then it makes sense to use their DID no?
Otherwise why look at their account if they can easily create new ones

Reply:
Yeah might make sense. Maybe something to discuss with Luca? I think it would be fine to store the Did in project_metadata/details and check for it in for example evaluation. At the same time, I don't think we need it here, right? As it actually only starts evaluation and does not do an evaluation?

Reply:
agreed, I created a Linear issue for this, but as mentioned this place does not need the did